app/views/sites/displays/_govboxes.html.haml

```diff
@@ -23,7 +23,7 @@
     - if @site.agency.present?
       %tr
         %td Job openings
-        %td.cell-1x USAJobs/NeoGov
```

Comment:
@peggles2 , per our conversation, this should be the only change in this commit. https://github.com/GSA/search-gov/pull/159 contains all the rest of the NeoGov code cleanup.

Also, let's change `USAJobs` to `USAJOBS`, as that is the correct capitalization.